TestModule/CompatibilityTest.c

```diff
@@ -1,4 +1,4 @@
 // Tests if the external inteface has basic compatibility with C compiler. 
 // IE compiles without errors and warnings.
 
-#include "op2ext.h"
\ No newline at end of file
+#include "include/op2ext.h"
```

Comment:
Having the `include/` folder explicitly in the `#include` is probably not ideal. I think usually an include folder setting would be added to the project.

Reply:
I'm happy either way. 

-Brett